### ISSUE TITLE
Not loading custom css file via content_css property.

### ISSUE THREAD

**Issue by seaniemc** [tinymce/tinymce-angular#115]
I am using the below versions
`"@tinymce/tinymce-angular": "^3.3.1", "tinymce": "^5.0.16", "@angular/core": "7.2.0",`

**How do I load custom CSS file to use with content_css property?**
I have been searching around the web, but I can't seem to get the answer to this question or at least I couldn't understand it. 

So I have a public editorConfig obj which gets called in the init function of the HTML

```
<editor 
        [init]="editorConfig"
        [(ngModel)]="articleObj.text" name="text" id="text"
        (ngModelChange)="onChangeWordCount()">
</editor >
```
```
public editorConfig = {
    plugins: 'lists advlist image code media link table wordcount',
    menubar: false,
    statusbar: false,
    branding: false,
    browser_spellcheck: true, 
    content_css: 'src/assets/content.css',

```
This loads TinyMCE and my editor work great. I followed the docs and the scripts load with the lazy loading. 

However, I am not able to load my src/assets/content.css file. I keep getting a 404. 
```
"assets": [
    "src/favicon.ico",
     "src/assets",
      { "glob": "**/*", "input": "node_modules/tinymce/skins", "output": "/skins/" },
      { "glob": "**/*", "input": "node_modules/tinymce/themes", "output": "/themes/" },
      { "glob": "**/*", "input": "node_modules/tinymce/plugins", "output": "/plugins/" }
 ],
```
I have added the path to my `angular.json` file
```
"styles": [
    "src/styles.scss",
   "src/assets/content.css"
],
```
Also would be good if somewhere in the documentation there was a reference to how this could be done.

**Comment by SimonFc** [comment]
Check the structure of your dist folder(or wherever your build output is). You'll probably see that your content.css file is at the root of that folder while in your code you assume that it is at `src/assets/content.css`. If you want to keep that structure have 
`{ "glob": "**/*", "input": "src/assets", "output": "/src/assets" }` in your assets property in the angular.json file instead of `"src/assets"`.

The ContentCss can't be in your "styles" property btw since that would just make the styles global.  TinyMCE must load it since it must be injected into TinyMCE's iframe.

Hope this helps!

**Comment by seaniemc** [comment]
@SimonFc 
I have updated the angular.json as you suggested. 
```
"assets": [
     "src/favicon.ico",
      "src/assets",
       { "glob": "**/*", "input": "node_modules/tinymce/skins", "output": "/skins/" },
       { "glob": "**/*", "input": "node_modules/tinymce/themes", "output": "/themes/" },
       { "glob": "**/*", "input": "node_modules/tinymce/plugins", "output": "/plugins/" },
       { "glob": "**/*", "input": "src/assets", "output": "/src/assets" }
  ]
```
Then in the editor component I am updating `content_css` property as: 
`content_css: 'src/assets/content.css',`

Now the file is getting loaded and I am no longer getting a 404. 
However, my CSS code doesn't seem to update the toolbar as I wanted. The change I was hoping for, was to change the toolbar background color. 

**CSS Code**
```
/* toolbar */
.mce-toolbar-grp {
    background-color:#efeeee !important; /* uses !important or override .mce-panel background-color/image */
    background-image: none !important;
}
```
[Which i got from here](https://stackoverflow.com/questions/24642448/changing-background-toolbar-colour-and-text-colour-in-tinymce-editor/24642595)

In short, I am not sure if my CSS is wrong or is the configuration/loading of the file wrong. 

Thanks again,
Sean.

**Comment by SimonFc** [comment]
Ah okay I see.

ContentCss isn't what you're looking for then as it's only for styling the content inside the iframe so ignore the `content_css` option. The UI is outside the iframe and to override the styling for that you'll need to include that CSS on the page. Actually to do that you would include it in your "styles" property after all(since these stylesheets will be injected into the header of your page)

Overriding styles is not recommended though since it'll break easily if you upgrade TinyMCE. If you wish to create your own skin the recommended way is to follow these [instructions](https://www.tiny.cloud/docs/advanced/creating-a-skin/#makingoreditingaskin). In the very near future we'll have a skin customizer tool for TinyMCE 5 which will make the process of creating skins a lot easier.
